test/unit/spec/EnrollDuo_spec.js

```diff
@@ -36,10 +37,6 @@ function (Okta, Duo, OktaAuth, Util, Beacon, Expect, Form, Router, $sandbox,
       Util.mockRouterNavigate(router, startRouter);
       Util.mockDuo();
       return tick()
```

Comment:
Q is not used. Was your intentional to replace `tick` with `Q`. Any chance to not even use Q?